canonical/juju-dashboard
# WD-5747 - Filter logs by the current model model

## Done

- Filter by model and hide model filter when viewing model audit logs.

## QA

- Connect to JAAS (not staging - this is so you can get to a model page).
- Open your network tab in your dev tools and find the controller websocket connection (it should be the one with the JIMM requests).
- Go to the logs page.
- In your controller websocket you should see the `FindAuditEvents` with no `model` param.
- Click the filter button.
- In the filter panel you should see the model name input.
- Enter a model name and click the Filter button.
- In your controller websocket you should see the `FindAuditEvents` with the `model` param set to the value you entered.
- Go to the audit logs tab inside a model.
- In your controller websocket you should see the `FindAuditEvents` with the `model` param set to the current model.
- Open the filter panel and there shouldn't be a model input.

## Details

https://warthogs.atlassian.net/browse/WD-5747
https://warthogs.atlassian.net/browse/WD-5222
